clippy_lints/src/utils/conf.rs

```diff
@@ -8,6 +8,13 @@ use std::error::Error;
 use std::path::{Path, PathBuf};
 use std::{env, fmt, fs, io};
 
+/// Holds information used by `IMPORT_RENAME` lint.
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Hash)]
```

Comment:
```suggestion
#[derive(Clone, Debug, Deserialize)]
```

nit: probably don't need all of these.